mosip_master/data_upgrade/1.1.5.5_to_1.2.0.1/data-uploader.py

```
@@ -114,15 +120,17 @@ def getAccessToken():
     'request': {
-      'appId': 'admin',
-      'password': password,
-      'userName': username
+      'appId': appId,
+      'clientId': clientId,
+      'secretKey': secretKey
     },
     'requesttime': getCurrentDateTime(),
     'version': 'string'
   }
   authresponse=requests.post(authURL, json= auth_req_data)
```

Ruff (0.15.17)

[error] Probable use of `requests` call without timeout

(S113)